cpu/esp32/Makefile.features

```diff
@@ -3,6 +3,7 @@
 include $(RIOTCPU)/esp_common/Makefile.features
 
 FEATURES_PROVIDED += arch_esp32
+FEATURES_PROVIDED += esp_wifi_ap
```

Comment:
For historical reasons, we have some inconsistencies where to define the features. While features `esp_wifi` and `esp_now` are defined in `boards/common/esp*/Makfefile.features`, feature `esp_wifi_enterprise` is defined here. So I'm thinking about whether it would make sense, to move `esp_wifi` and `esp_now` to `cpu/esp_common/Makfefile.features` piggybacked with this PR. I'm not really sure at the moment.

Reply:
I'm a little bit confused here, should `esp_wifi_ap` feature (and `esp_wifi`/`esp_now`) be in `cpu/esp_common/Makefile.features` right? I first put the feature only on the `cpu/esp32/Makefile.features` because it was the only I supported/tested (and partly because `esp_wifi` feature wasn't on `cpu/esp_common/Makefile.features`).

Reply:
> So I'm thinking about whether it would make sense, to move `esp_wifi` and `esp_now` to `cpu/esp_common/Makfefile.features` piggybacked with this PR.

Oh sorry, I was already thinking about the next step :sunglasses: 

```suggestion
```
Please move this to `cpu/esp_common/Makefile.features` since it is valid for ESP32 and ESP8266. For the `esp_wifi_enterprise`, the situation differs because it is only supported on ESP32.

Moving `esp_wifi` and `esp_now` from `boards/common/esp*/Makfefile.features` to `cpu/esp_common/Makfefile.features` would be the next step.

Reply:
Solved then :wink: